.github/workflows/build_and_test.yml

```diff
@@ -30,30 +30,20 @@ jobs:
       with:
         fetch-depth: '0'
     - run: git fetch --depth=1 origin +refs/tags/*:refs/tags/* +refs/heads/*:refs/remotes/origin/*
-    - name: Set up Python 3.9
+    - name: Set up Python 3.10
       uses: actions/setup-python@v5
       with:
-        python-version: '3.9'
-    - name: Build and test including remote checks (3.9) mypy
+        python-version: '3.10'
+    - name: Build and test including remote checks (3.10) mypy
       if:  (matrix.os == 'macos-12') && (github.event_name == 'push' || (github.event_name == 'pull_request' &&  github.event.pull_request.head.repo.full_name == github.repository) || github.event_name == 'release' || github.event_name == 'schedule' )
       shell: bash
       run: |
         ./.github/workflows/build-test mypy
-    - name: Build and test including remote checks (3.9) nomypy
+    - name: Build and test including remote checks (3.10) nomypy
       if:  (matrix.os != 'macos-12') && (github.event_name == 'push' || (github.event_name == 'pull_request' &&  github.event.pull_request.head.repo.full_name == github.repository) || github.event_name == 'release' || github.event_name == 'schedule')
       shell: bash
       run: |
         ./.github/workflows/build-test nomypy
-    - name: Set up Python 3.10
```

Comment:
Can we restore this? And use python 3.12 for the PR workflows...

Reply:
In terms of the "mapping", I previously did 3.9->3.10 & dropped 3.10.
I've now done 3.9->3.10 & 3.10->3.11/3.12.

Is this what you meant? Sorry if "mapping" obfuscates things

Reply:
Yeah that sounds good.